src/sql/workbench/parts/notebook/test/browser/notebookEditorModel.test.ts

```diff
@@ -705,4 +869,47 @@ suite('Notebook Editor Model', function (): void {
 		await textFileEditorModel.load();
 		return new NotebookEditorModel(defaultUri, textFileEditorModel, mockNotebookService.object, accessor.textFileService, testResourcePropertiesService);
 	}
+
+	function setupTextEditorModelWithEmptyOutputs(notebookEditorModel: NotebookEditorModel, newCell: ICellModel) {
+		// clear outputs
+		newCell[<any>'_outputs'] = [];
+
+		let contentChange: NotebookContentChange = {
+			changeType: NotebookChangeType.CellsModified,
+			cells: [newCell],
+			cellIndex: 0
+		};
+
+		notebookEditorModel.updateModel(contentChange, NotebookChangeType.CellsModified);
+		should(notebookEditorModel.lastEditFullReplacement).equal(true);
```

Comment:
Definitely add a message here indicating what the problem is. Generally all non-string comparisons should have that (since it's difficult to determine what the issue is from the assert failure alone) and IMO it's worth it to put for string comparisons as well.